### ISSUE TITLE
Install Sage optional package giacpy_sage

### ISSUE THREAD

**Issue by slel** [sagemathinc/cocalc#3008]
Please install giacpy_sage so that Sage can use giac to compute Gröbner bases.

- Homepage of giacpy and giacpy_sage: http://webusers.imj-prg.fr/~frederic.han/xcas/giacpy/
- Code repository: https://gitlab.math.univ-paris-diderot.fr/han/giacpy-sage
- Release tarballs: http://webusers.imj-prg.fr/~frederic.han/xcas/giacpy/sage/

To install:
```
sage -i giacpy_sage
```

Testing that this makes libgiac work:
```
sage: from giacpy_sage import libgiac, giacsettings
sage: x, y, z = libgiac('x,y,z')    # add some giac objects
sage: f=(x+3*y)/(x+z+1)^2 -(x+z+1)^2/(x+3*y)
sage: f.factor()
(3*y-x^2-2*x*z-x-z^2-2*z-1)*(3*y+x^2+2*x*z+3*x+z^2+2*z+1)/((x+z+1)^2*(3*y+x))
sage: f.normal()
(-x^4-4*x^3*z-4*x^3-6*x^2*z^2-12*x^2*z-5*x^2+6*x*y-4*x*z^3-12*x*z^2-12*x*z-4*x+9*y^2-z^4-4*z^3-6*z^2-4*z-1)/(x^3+3*x^2*y+2*x^2*z+2*x^2+6*x*y*z+6*x*y+x*z^2+2*x*z+x+3*y*z^2+6*y*z+3*y)
```
Testing that this allows to compute Gröbner bases with giac in Sage:
```
sage: R.<x, y, z> = PolynomialRing(QQ, 3)
sage: J = Ideal([x^2 + y + z - 1, x + y^2 + z - 1, x + y + z^2 - 1])
sage: B = J.groebner_basis(algorithm='giac:gbasis')
```

**Comment by haraldschilly** [comment]
available for quite a while

![Screenshot from 2020-09-10 10-40-29](https://user-images.githubusercontent.com/207405/92702950-14335e00-f352-11ea-86d9-8b39080a735a.png)

**Comment by slel** [comment]
Thanks!

Note: from Sage 9.2, `giacpy_sage` is merged into Sage,
so it no longer needs to be installed on top of Sage.

This is thanks to 

- [Sage Trac ticket 29171: Move giacpy_sage into sage source code](https://trac.sagemath.org/ticket/29171)

(merged in Sage 9.2.beta10).
